transactions/add-access-nodes/2024/sept-24/arguments.json

```diff
@@ -0,0 +1,6 @@
+[
+    {
+        "type": "UInt16",
+        "value": "10"
```

Comment:
In the README, you say you want to increase the slot limit for access nodes by 6, but here you're increasing it by 10. Which one should it be?

Reply:
fixed it to 10